include/swift/Option/Options.td

```diff
@@ -466,4 +466,8 @@ def sanitize_EQ : CommaJoined<["-"], "sanitize=">,
   Flags<[FrontendOption, NoInteractiveOption]>, MetaVarName<"<check>">,
   HelpText<"Turn on runtime checks for erroneous behavior.">;
 
+def sanitize_coverage_EQ : Joined<["-"], "sanitize-coverage=">,
+  Flags<[FrontendOption, NoInteractiveOption]>,
+  MetaVarName<"<coverage_type>">, HelpText<"Specify sanitize coverage type">;
```

Comment:
> At the very least it should be "sanitizer coverage", right?
Oops. Yes it should be.

The help text for Clang's ``-fsanitize-coverage=`` is "Specify the type of coverage instrumentation for Sanitizers". Should I just use that?